packages/data-driver-postgres/package.json

```diff
@@ -23,22 +23,26 @@
 	"scripts": {
 		"build": "tsup src/index.ts --format=esm --dts",
 		"dev": "tsup src/index.ts --format=esm --dts --watch",
-		"test": "vitest --watch=false"
+		"test": "vitest --watch=false",
+		"depcruise": "depcruise src --include-only '^src' -x test.ts --output-type dot | dot -T svg > dependency-graph.svg"
```

Comment:
Hm I like it there because it makes it so easy to have a look at this. Otherwise one would install it themselves, add the config, find out the command and would repeat that the next time he/she wants to have a look. with this included the hurdle would not that big to have a look. I could have a look at this solely, but then I would either do the steps just mentions each time, or leave it locally the whole time which is also a bit annoying 🤔

Reply:
I think it's fine to have it included as long as it's a devDeps 😄 We can always remove / move it away later on once the project is out of dev state.